ai_agents/job_posting_agent/app.py

```
@@ -0,0 +1,125 @@
+        1. **Research Analyst** — searches DuckDuckGo for company culture, values, and news
+        2. **Job Requirements Specialist** — defines skills and qualifications for the role
+        3. **Job Posting Writer** — produces a polished, tailored job posting
+
+        Powered by **DeepSeek V4 Flash** via NVIDIA NIM.
+        """
+    )
+    st.markdown("---")
+    st.caption("Set `NVIDIA_API_KEY` in your `.env` file to run.")
+    st.caption("Watch the terminal running Streamlit for step-by-step logs.")
+    st.caption(
+        "First NVIDIA request may take 30–60s while the model warms up "
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)